e2e/motoko/Main.mo

```diff
@@ -109,7 +109,7 @@ shared ({ caller = installer }) actor class Main() {
             };
 
             let candidRpcCycles = 1_000_000_000_000;
-            let ethMainnetSource = #EthMainnet(?[#Ankr, #BlockPi, #Cloudflare, #PublicNode]);
+            let ethMainnetSource = #EthMainnet(?[#Ankr, #Cloudflare, #PublicNode]);
```

Comment:
Why is BlockPi removed?

Reply:
Was causing intermittent CI issues due to the response size limits.